web/src/main/java/io/fabric8/launcher/web/producers/GitServiceProducer.java

```diff
@@ -61,11 +68,12 @@ GitService getGitService(final HttpServletRequest request,
         GitServiceConfig gitServiceConfig = getGitServiceConfig(request);
         GitServiceFactory gitServiceFactory = gitServiceFactories.getGitServiceFactory(gitServiceConfig.getType());
         String provider = gitServiceFactory.getName().toLowerCase();
-        String gitAuthHeader = request.getHeader(GIT_AUTHORIZATION_HEADER);
+        String header = request.getHeader(GIT_AUTHORIZATION_HEADER);
+        String gitAuthHeader = tokenProvider.decryptToken(removeBearerPrefix(header));
```

Comment:
This breaks existing clients that send the Git token directly in the request. I'd prefer to add a new header indicating that the Bearer token needs to be decrypted before using it (something like `X-Git-Authorization-Encrypted:true`)

Reply:
removed this completly